### PR TITLE
Fix for #97 : CustomerGetWorkRequestsController/Logging

### PR DESCRIPTION
**Pull request by vendra131**
Fix for #97 : Added required loggers and 9 tests for same.

### CODE REVIEW

File: src/main/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsController.java [hunk 1]
```diff
@@ -24,24 +27,45 @@
...
+    if (sessionUser.getId() != user.get().getId()){
+      loggerService.warn(
+              WORK_REQUEST_ERROR,
+              UNAUTHORIZE_GET_WORK_REQUESTS_FOR_USER,
+              FAILURE
+      );
       throw new ValidationException(UNAUTHORIZE_GET_WORK_REQUESTS_FOR_USER);
+    }
+
+    loggerService.info(WorkRequestConstants.FIND_WORK_REQUEST_BY_CUSTOMER, ownerId);
```
Comment (magwas) [inline src/main/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsController.java:62]: this is unnecessary

File: src/main/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsController.java [hunk 2]
```diff
@@ -24,24 +27,45 @@
...
 
     final List<WorkRequestEntity> requests =
         workRequestRepository.findByCustomer(user.get());
 
+    loggerService.debug(WorkRequestConstants.FIND_WORK_REQUEST_BY_CUSTOMER_STATUS, ownerId, SUCCESS);
```
Comment (magwas) [inline src/main/java/com/kodekonveyor/work_request/CustomerGetWorkRequestsController.java:67]: this is unnecessary

**Comment by vendra131** [comment]
@magwas Removed the unnecessary logging as requested and updated PR

**Comment by szirbucz** [comment]
testcases=5
please fill in https://docs.google.com/forms/d/e/1FAIpQLSdNMbUDPHp2CCbsKtx3qMDCGXIYmmRsjJoF201Xtl-xvb2llQ/viewform

**Pull request merged**